testdata/taskrouter_responses.go

```diff
@@ -15,3 +15,31 @@ var TaskRouterCreateResponse = []byte(`
     "workspace_sid": "WS7a2aa7d8acc191786ad3c647c5fc3110"
 }
 `)
+
```

Comment:
This is great - can we delete the other response in responses_test.go and repoint references here?